nautobot/extras/forms.py

```diff
@@ -747,6 +749,26 @@ def requires_input(self):
         """
         return bool(len(self.fields) > 1)
 
+    def get_serializeable_data(self):
```

Comment:
To streamline the interface and not having to jump back between `JobForm` and `Job, I think this should move to `BaseJob.get_serializeable_data()` and then take a form's `cleaned_data` as an input.